### ISSUE TITLE
Change the color scheme

### ISSUE THREAD

**Issue by dhairyaj** [Documentive/TemplateBuddy-Old#5]

**Comment by dollyjain51099** [comment]
I want to work on this issue can u please assign it to me.

**Comment by dhairyaj** [comment]
Sure @dollyjain51099 , will assign this to you.

**Comment by dollyjain51099** [comment]
Can you brief me about the issue! Shall i add a toggle button to change the theme from dark to light and vice-versa?

**Comment by dhairyaj** [comment]
Sure.
The first thing you can do is tweaking the existing theme. It's kind of single color but the monotone is not catchy. The current color is pink theme with skyblue active tabs. You can modify it to another theme if you like.
Apart from this, a dark theme and dark-light theme toggle button is also needed. The toggle button can be provided near the logout button.

**Comment by dollyjain51099** [comment]
okay, so for dark theme and dark-light theme toggle button can i create an issue and work on it?

**Comment by dhairyaj** [comment]
you dont want to work on changing the base theme?

**Comment by dollyjain51099** [comment]
yeah, currently will be working on the base theme and on the later part create an issue and work on toggle button

**Comment by dollyjain51099** [comment]
okay

**Comment by dollyjain51099** [comment]
sure!

**Comment by dhairyaj** [comment]
After completing this issue, you can create a new issue for the dark theme-light theme toggle button.

**Comment by dollyjain51099** [comment]
okay!

**Comment by dollyjain51099** [comment]
![theme](https://user-images.githubusercontent.com/54888610/102614495-58f14100-415a-11eb-9608-d8381569f1fe.png)
Please check whether the color theme is fine!

**Comment by frankhart2018** [comment]
@dollyjain51099 I don't think having so many different colors looks good. I guess the login/signup UI looks good as is, this issue mostly focusses on the UI for the pages which are presented to the user after he/she logs in :).

**Comment by frankhart2018** [comment]
Created issue for dark-light theme toggle button. #105. Please post a comment on that, GitHub doesn't allow assignees who haven't posted a comment on the issue unless the person is a core dev.

**Comment by dollyjain51099** [comment]
Sure!
